test/shaderdb/ObjInput_TestIndexingInterpOfInputArray_lit.frag

```diff
@@ -37,35 +37,47 @@ void main()
 ; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIR-V lowering results
-; SHADERTEST: call <4 x float> @llpc.input.import.generic.v4f32{{.*}}
-; SHADERTEST: call <2 x float> @llpc.input.interpolate.evalij.offset.v2f32
-; SHADERTEST: call <4 x float> @llpc.input.import.interpolant.v4f32{{.*}}v2f32
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline before-patching results
+; SHADERTEST: = call <3 x float> @llpc.input.import.builtin.InterpPullMode.v3f32.i32(i32 268435459)
+; SHADERTEST: = call i32 @llvm.amdgcn.mov.dpp.i32(i32 %{{.*}}, i32 245, i32 15, i32 15, i1 true)
```

Comment:
It seems there are much more mov.dpp after your change. Is it missed in previous pattern or  due to your code change? Thanks!

Reply:
I have just checked, and the test generates 12 dpps both before and after my change. They were not in that part of the old pattern because the evalij has not been expanded yet.